src/pages/overview/pdf-accessibility-auto-tag-api/gettingstarted.md

```diff
@@ -0,0 +1,368 @@
+
+<InlineAlert slots="text" />
+Other sample programs in the SDK let you try out API options including generating a tagging report, and shifting headings so that the document title is only H1.
+
+<InlineAlert slots="text" />
+
+Command line execution is not mandatory. You can import the samples Maven project into your preferred IDE (e.g. IntelliJ/Eclipse) and run the samples from there.
+
+#### Verifying download authenticity
+
+For security reasons you may wish to confirm the installer's authenticity. To do so,
+
+1.  After installing the package, navigate to the `.jar.sha1` file.
+2.  Calculate the hash with any 3rd party utility.
+3.  Find and open PDF Services sha1 file. Note: if you're using Maven, look in the .m2 directory.
+4.  Verify the hash you generated matches the value in the .sha1 file.
+
+```
+298bf24b42e543029382a624b9e2c1f0069d90a5
+```
+
+#### Logging
+
+Refer to the API docs for error and exception details.
+
+-   For logging, use the [slf4j API](https://www.slf4j.org/) with a log4js-slf4j binding.
+-   Logging configurations are provided in src/main/resources/log4js.properties.
+-   Specify alternate bindings, if required, in pom.xml.
+
+**log4js.properties file**
+
+```properties
+name=PropertiesConfig
+appenders = console
+
+# A sample console appender configuration, Clients can change as per their logging implementation
+rootLogger.level = WARN
+rootLogger.appenderRefs = stdout
+rootLogger.appenderRef.stdout.ref = STDOUT
+
+appender.console.type = Console
+appender.console.name = STDOUT
+appender.console.layout.type = PatternLayout
+appender.console.layout.pattern = [%-5level] %d{yyyy-MM-dd HH:mm:ss.SSS} [%t] %c{1} - %msg%n
+
+loggers = pdfservicessdk,validator,apache
+
+# Change the logging levels as per need. INFO is recommended for pdfservices-sdk
+logger.pdfservicessdk.name = com.adobe.pdfservices.operation
+logger.pdfservicessdk.level = INFO
+logger.pdfservicessdk.additivity = false
+logger.pdfservicessdk.appenderRef.console.ref = STDOUT
+
+logger.validator.name=org.hibernate
+logger.validator.level=WARN
+
+logger.apache.name=org.apache
+logger.apache.level=WARN
+```
+   
+
+![Samples directory structure Java](../images/samplefilesjava.png)
+
+#### Custom projects
+
+While the samples use Maven, you can use your own tools and process.
+
+To build a custom project:
+
+1.  Access the .jar in the [central Maven repository](https://www.adobe.com/go/pdftoolsapi_java_maven).
+2.  Use your preferred dependency management tool (Ivy, Gradle, Maven), to include the SDK .jar dependency.
+3.  Open the pdfservices-api-credentials.json downloaded when you created your credential.
+4.  Add the [Authentication](./index.md#authentication) details as described above.
+
+![Adobe PDF Services SDK On Maven](../images/maven.png)
+
+### Python
+
+Jump start your development by bookmarking or downloading the following key resources:
+
+-   This document
```

Comment:
These links are pointing to beta versions currently, which should be updated to main/master once development is complete.